### PR TITLE
purge xcode project from cache after updating it #33

### PR DESCRIPTION
**Pull request by Fender123**

### CODE REVIEW

**Comment by rodrigograca31** [comment]
@Fender123 did you test it? does it work well?

File: scripts/create_ios_strings.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 var fs = require('fs-extra');
 var _ = require('lodash');
 var iconv = require('iconv-lite');
-var xmldom = require('xmldom');
+var xmldom = require('xmldom');
```
Comment (rodrigograca31) [inline scripts/create_ios_strings.js]: Seems `xmldom` is not used.......?

Comment (rodrigograca31) [reply scripts/create_ios_strings.js]: Oh! Seems you didnt touch `xmldom`.... it was `path`...

**Comment by Fender123** [comment]
> @Fender123 did you test it? does it work well?

yes, it is working fine.

**Comment by rodrigograca31** [comment]
Thanks for contributing @Fender123 :heart:

**Pull request merged**